addOns/alertFilters/src/main/resources/org/zaproxy/zap/extension/alertFilters/resources/Messages.properties

```diff
@@ -1,14 +1,69 @@
-alertFilters.api.action.addAlertFilter = Adds a new alert filter for the context with the given ID. 
-alertFilters.api.action.addGlobalAlertFilter = Adds a new global alert filter. 
+alertFilter.api.desc = Test
```

Comment:
Needs remove.

Reply:
Oops, thanks. That was leftover from when I was trying to sort out prefixes and the top desc.